api/src/main/scala/hmda/api/http/ValidationErrorConverter.scala

```diff
@@ -62,13 +70,29 @@ trait ValidationErrorConverter {
       .getOrElse("")
   }
 
-  private def relevantFields(err: ValidationError): JsObject = {
+  private def relevantRow(err: ValidationError, vs: HmdaFileValidationState): HmdaFileRow = {
+    if (err.ts) vs.ts.get
+    else vs.lars.find(lar => lar.loan.id == err.errorId).get
+  }
```

Comment:
Good point. when the code gets to this point, the `HmdaFileValidationState` _should_ include the TS and LARs, but that doesn't mean it always will. 

I added code to keep it from failing in e1b81efb8e7a3c819aa00ea58b8d11a7e3dd98b6.